python/tchannel/messages/call_continue.py

```diff
@@ -0,0 +1,94 @@
+from __future__ import absolute_import
+
+from . import common, ChecksumType
+from .base import BaseMessage
+from .common import FlagsType
+
+
+class CallContinueMessage(BaseMessage):
+    """Represent a continuation of a call request (across multiple frames)."""
+    max_args_num = 3
+
+    __slots__ = (
+        'flags',
+        'checksum',
+        'args',
+    )
+
+    def __init__(
+        self,
+        flags=0,
+        checksum=None,
+        args=None,
+    ):
+        self.flags = flags
+        if checksum is not None:
+            checksum = common.ChecksumType.standardize(checksum)
+        self.checksum = checksum or \
+            (common.ChecksumType.none, None)
+
+        self.args = args or []
+
+    def encode(self):
+        self.args = list(map(
+            lambda arg: common.encode_str(arg),
+            self.args))
+
+    def decode(self):
```

Comment:
I don't understand why you're doing the UTF-8 encode/decode. As far as TChannel is concerned, arg1, 2, and 3 are binary blobs. If they are unicode strings, the caller is expected to encode them before passing them in.